<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenuWindow</class>
 <widget class="QWidget" name="MainMenuWindow">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>800</width><height>600</height></rect>
  </property>
  <property name="minimumSize">
   <size><width>800</width><height>600</height></size>
  </property>
  <property name="windowTitle">
   <string>Main Menu</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>ðŸ“š PERSONAL LIBRARY</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 24px; font-weight: bold;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bodyLayout">
     <item>
      <layout class="QVBoxLayout" name="navLayout">
       <item>
        <widget class="QPushButton" name="booksButton">
         <property name="text"><string>ðŸ“– Books</string></property>
         <property name="styleSheet"><string notr="true">font-weight: bold;</string></property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="readingListsButton">
         <property name="text"><string>ðŸ“‹ Reading Lists</string></property>
         <property name="styleSheet"><string notr="true">font-weight: bold;</string></property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="goalsButton">
         <property name="text"><string>ðŸŽ¯ Goals</string></property>
         <property name="styleSheet"><string notr="true">font-weight: bold;</string></property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="accountButton">
         <property name="text"><string>ðŸ‘¤ Account</string></property>
         <property name="styleSheet"><string notr="true">font-weight: bold;</string></property>
        </widget>
       </item>
       <item>
        <spacer name="navSpacer">
         <property name="orientation"><enum>Qt::Vertical</enum></property>
         <property name="sizeType"><enum>QSizePolicy::Expanding</enum></property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="logoutButton">
         <property name="text"><string>ðŸšª Logout</string></property>
         <property name="styleSheet">
          <string notr="true">font-weight: bold; color: red;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QStackedWidget" name="mainContent"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>